packages/core/upload/server/register.js

```diff
@@ -60,6 +62,25 @@ const createProvider = (config) => {
     );
   }
 
+  if (providerOptions?.sizeLimit) {
```

Comment:
this option being only document for the local provider I think we should put the warning in the local provider itself